pwndbg-lldb.py

```diff
@@ -105,12 +105,24 @@ def find_lldb_python_path() -> str:
     if len(sys.argv) == 2:
         target = sys.argv[1]
 
+    from pwndbg.dbg.lldb.repl import PwndbgController
     from pwndbg.dbg.lldb.repl import run as run_repl
 
     if debug:
         print("[-] Launcher: Entering Pwndbg CLI")
 
-    run_repl([f"target create '{target}'"] if target else None, debug=debug)
+    def drive(startup: List[str] | None):
+        async def drive(c: PwndbgController):
+            if startup is not None:
+                for line in startup:
+                    await pwndbg.execute_and_capture(line)
+
+            while True:
+                await pwndbg.interactive()
+
+        return drive
```

Comment:
```suggestion
    def drive(startup: List[str] | None):
        async def drive(c: PwndbgController):
            if startup is not None:
                for line in startup:
                    await c.execute_and_capture(line)

            while True:
                await c.interactive()

        return drive
```
just to fix other `pwndbg` names, that previously I missed

Reply:
Ended up making this outdated, so I fixed them manually.